site/_data/lib/date.js

```diff
@@ -1,4 +1,13 @@
-const formatDate = (date, locale = 'en', opts) => {
+/**
+ * Returns a formatted date string.
+ * See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat
+ * for a list of options.
+ * @param {Date} date
+ * @param {string} [locale="en"]
+ * @param {object} opts
```

Comment:
Could mark this as optional as well...

```js
 * @param {object} [opts]
 ```

Reply:
FWIW, TS doesn't need this hint when it sees a `=` in the function itself.